src/io/conekta/Resource.java

```diff
@@ -82,7 +82,41 @@ protected static ConektaObject scpWhere(String className, JSONObject params) thr
         resource.loadFromArray(jsonArray);
         return resource;
     }
-    
+
```

Comment:
espacios en blanco inútiles

Reply:
Este no aplica  ya que que agrego un solo espacio